packages/cyberstorm/src/components/CommunityCard/CommunityCardSkeleton.module.css

```diff
@@ -0,0 +1,44 @@
+.root {

```

Comment:
Just wondering:

- Are all of the style declarations in this file needed? E.g. overflow, transition, text-overflow, font stuff. It's not like the cards are going to have content?
- Is the style file needed at all, or could we perhaps reuse the styles from the actual component? Note that this is just a thought and IDK if it's doable at all. Anyways, would save us the trouble of updating the skeleton if the actual community card is updated.

Reply:
You are on point that here, some of stuff is unnecessary and was intended to be scraped.
Made this bit raw draft PR so @Oksamies  could checkout/correct if logic would be now made expected way.
Having said that these points have been helpful once again 👍